database/seedFunctions.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable no-await-in-loop */
+const axios = require('axios');
+
+const generateRandomPercentage = () => (Math.floor(Math.random() * 100) / 100);
+
+const generateNumberWithinRange = (min, max) => (Math.floor(Math.random() * (max - min) + min));
+
+const generateFillerText = async (options) => {
+  let text;
+  if (options.paras) {
+    text = await axios.get(`https://baconipsum.com/api/?type=meat-and-filler&paras=${options.paras}&format=text`);
+  } else if (options.sentences) {
+    text = await axios.get(`https://baconipsum.com/api/?type=meat-and-filler&sentences=${options.sentences}&format=text`);
```

Comment:
Nice touch with async/await , something we haven't learnt yet. 

Other option would be : Axios returns a `Promise` and use `then` block to return response in a `cb` to the calling function